core/arch/arm/kernel/user_ta.c

```diff
@@ -712,6 +747,58 @@ static TEE_Result add_deps(struct user_ta_ctx *utc __unused,
 
 #endif
 
+static TEE_Result register_ro_slices(struct file **file,
+				     const struct user_ta_store_ops *ta_store,
+				     struct user_ta_store_handle *handle,
+				     struct load_seg *segs, size_t num_segs)
+{
+	TEE_Result res = TEE_SUCCESS;
+	struct file *f = NULL;
+	struct file_slice *fs = NULL;
+	size_t num_slices = 0;
+	size_t n = 0;
+	uint8_t tag[FILE_TAG_SIZE];
```

Comment:
I guess here it is fine to not init `tag`. But that's against the rule.

Reply:
I'll fix